### DISCUSSION TITLE
Option to enable/disable specific submod?

### DISCUSSION THREAD

**Discussion by towfique-pranto** [Monika-After-Story/MonikaModDev#8270] [general - submod / external content]
I might be wrong but there aren't any options to enable/disable specific submod without removing them completely. I think it would be very convenient to have this feature for testing submods or if you want to stop a submod temporarily (Example: I gave my monika a new look and don't want the auto outfit changer to change it for few days but can't do so without removing it)

**Comment by Booplicate** [comment]
We do plan something like this, for now you'll have to uninstall the submods you don't like.
